charts/maxscale/templates/mariadb/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.mariadb.resources.limits.memory }}
+{{- $unit := regexSplit "([0-9]+(?:\".\"[0-9]+)?)" .Values.mariadb.resources.limits.memory -1 | last }}
+{{- $limit := round (mulf (trimSuffix $unit .Values.mariadb.resources.limits.memory) (0.8)) 3 | toString }}
```

Comment:
```
[ERROR] Unknown suffix '.' used for variable 'innodb-buffer-pool-size' (value '2.4Gi')
[ERROR] /opt/bitnami/mariadb/sbin/mysqld: Error while setting value '2.4Gi' to 'innodb-buffer-pool-size'
[ERROR] Parsing options for plugin 'InnoDB' failed. 
[ERROR] /opt/bitnami/mariadb/sbin/mysqld: unknown variable 'innodb_strict_mode=1'
[ERROR] Aborting
```

I also had to manually set the configMap's name to `mariadb.configurationConfigMap`.

Reply:
Note: Same Error with `2.4G`.